idaes/generic_models/flowsheets/methanol_flowsheet_w_recycle.py

```diff
@@ -11,471 +11,12 @@
 # at the URL "https://github.com/IDAES/idaes-pse".
 ##############################################################################
 """
-Task: IDAES Support for ARPE-E Differentiate
-Scenario: Methanol Synthesis From Syngas
-Author: B. Paul and M. Zamarripa
+Deprecation path for renamed module.
```

Comment:
As before, deprecation paths are only needed for things that were in the repo before this PR.